src/index.css

```
@@ -44,6 +44,68 @@ body {
+    transform: translate(3px, -1px);
+  }
+}
+
+@keyframes heart {
+  0%,
+  100% {
+    opacity: 0.5;
+  }
+  50% {
+    opacity: 1;
+    box-shadow: 0 0 9px currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)